api-metadata.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "definitions": {
+
+    "function_metadata": {
+      "type": "object",
+      "properties": {
+        "minArgs": {
+          "description": "The minimum number of arguments which must be passed to the function. If called with fewer than this number of arguments, the wrapper will raise an exception.",
+          "type": "number"
+        },
+        "maxArgs": {
+          "description": "The maximum number of arguments which may be passed to the function. If called with more than this number of arguments, the wrapper will raise an exception.",
+          "type": "number"
+        },
+        "fallbackToNoCallback": {
+          "description": "If the function doesn't take a callback parameter in some browsers and throws an error when a callback is passed.",
+          "type": "boolean"
+        },
+        "noCallback": {
+          "description": "Set when `fallbackToNoCallback` is set and this browser doesn't support callbacks for this function.\nMUST NOT BE SET IN api-metadata.json.",
+          "type": "boolean"
+        },
+        "singleCallbackArg": {
```

Comment:
The meaning of this changes in #175 , as documented here : https://github.com/mozilla/webextension-polyfill/issues/168#issuecomment-453623762